blocks/editable/test/index.js

```diff
@@ -40,4 +40,51 @@ describe( 'Editable', () => {
 		} );
 		/* eslint-enable no-console */
 	} );
+
+	describe( 'Editable.Value', () => {
+		it( 'should render null when null passed as value', () => {
+			const wrapper = shallow( <Editable.Value value={ [ null ] } /> );
+
+			expect( wrapper.html() ).toBe( null );
+		} );
+
+		it( 'should render null when false passed as value', () => {
+			const wrapper = shallow( <Editable.Value value={ [ false ] } /> );
```

Comment:
Is this needed? Not sure ho that could happen.

Reply:
(Same for the above.)

Reply:
We can drop it, I just wanted to track down why this fails in test env.